js/all.min.js

```diff
@@ -21,4 +21,16 @@ function mobileView () {
   const menu = document.querySelector('div.mobile-menu');
   menu.classList.toggle('hidden');
   menu.classList.toggle('flex');
-}
\ No newline at end of file
+}
+
+/* Table of contents */
+const tableOfContents = document.querySelector('div > div> article> div> #table-of-contents + ol');
```

Comment:
might be easier to pick `h2#table-of-contents`

Reply:
Yes, I will update it. I was copying that from dev tools just to be exact specific